### DISCUSSION TITLE
should be able to follow a community or a new source completely to see in my feed

### DISCUSSION THREAD

**Discussion by seeniforu** [dailydotdev/daily#764] [Feature Requests]
For example : there is this Hacker Noon posts. 
need a option to add that source completely to my feed.
There will be some other sources also. by adding that also, we can see it in my feed.
In my feed including other posts, now all hacker noon's posts should also be visible. 
To put it in short a feature to follow a community or a new source.

**Comment by idoshamun** [comment]
Hi @seeniforu,

Thanks for the suggestion. Just to clarify you want the ability to select several sources that will always be shown in your feed regardless of the tags?

**Reply by seeniforu** [reply]
yeah that's what i'm thinking. Everyone not going like that but you can give choice for sources to be shown in feed.
